lib/takeaway.rb

```diff
@@ -0,0 +1,44 @@
+require_relative 'menu'
+require_relative 'order'
+require_relative 'sms'
+class Takeaway
+
+  attr_reader :menu, :order
+
+  def initialize(menu_klass = Menu.new, order_klass = Order.new, sms_klass = Sms.new)
```

Comment:
Metrics/LineLength: Line is too long. [85/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)